Dockerfile

```diff
@@ -1,22 +1,14 @@
 FROM ocaml/opam:alpine
 
-# Workaround for https://github.com/docker/datakit/issues/152
-RUN cd /home/opam/opam-repository && git reset --hard dd38f0690ff01c8e1ed0eed231307ba4d00c50df && opam update
-RUN opam depext lwt ssl &&  opam install lwt alcotest oasis
-RUN opam pin add asl.dev https://github.com/mirage/ocaml-asl.git
-#RUN cd /home/opam/opam-repository && git pull && opam update
+RUN sudo apk add ncurses-dev
+RUN opam depext lwt && opam install lwt alcotest
 
 RUN opam pin add hvsock https://github.com/djs55/ocaml-hvsock.git
 
-COPY opam /home/opam/src/datakit/opam
```

Comment:
I am not sure we want to use the Dockerfile in dev mode. I think keeping these files simple is better.